### PR TITLE
Newsletter-222: add Chinese translation

### PR DESCRIPTION
**Pull request by hulatown**

### CODE REVIEW

File: _posts/zh/newsletters/2022-10-19-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+[brqgoo]: https://twitter.com/brqgoo/status/1579216353780957185
+[big msig]: https://blockstream.info/tx/7393096d97bfee8660f4100ffd61874d62f9a65de9fb6acf740c4c386990ef73?expand
+[news65 tapscript limits]: /en/newsletters/2019/09/25/#tapscript-resource-limits
+[lnd 0.15.2-beta]: https://github.com/lightningnetwork/lnd/releases/tag/v0.15.2-beta
+[news221 lnd]: /en/newsletters/2022/10/12/#lnd-v0-15-2-beta
+[btcd 0.23.2]: https://github.com/btcsuite/btcd/releases/tag/v0.23.2
+[verloren limits]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/020993.html
+[sanders limits]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/020996.html
+[news205 rbf]: /en/newsletters/2022/06/22/#full-replace-by-fee
+[news208 rbf]: /en/newsletters/2022/07/13/#bitcoin-core-25353
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2022-10-19-newsletter.md:123]:
```suggestion
[news208 rbf]: /zh/newsletters/2022/07/13/#bitcoin-core-25353
```

File: _posts/zh/newsletters/2022-10-19-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+[superset rbf]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2013-March/002240.html
+[brqgoo]: https://twitter.com/brqgoo/status/1579216353780957185
+[big msig]: https://blockstream.info/tx/7393096d97bfee8660f4100ffd61874d62f9a65de9fb6acf740c4c386990ef73?expand
+[news65 tapscript limits]: /en/newsletters/2019/09/25/#tapscript-resource-limits
+[lnd 0.15.2-beta]: https://github.com/lightningnetwork/lnd/releases/tag/v0.15.2-beta
+[news221 lnd]: /en/newsletters/2022/10/12/#lnd-v0-15-2-beta
+[btcd 0.23.2]: https://github.com/btcsuite/btcd/releases/tag/v0.23.2
+[verloren limits]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/020993.html
+[sanders limits]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/020996.html
+[news205 rbf]: /en/newsletters/2022/06/22/#full-replace-by-fee
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2022-10-19-newsletter.md:122]:
```suggestion
[news205 rbf]: /zh/newsletters/2022/06/22/#policy-limits
```

File: _posts/zh/newsletters/2022-10-19-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="23549,25412,25667,2448,6956,6972,7004,1625,26323,1372,1378,26265" %}
+[bitcoin core 24.0 rc2]: https://bitcoincore.org/bin/bitcoin-core-24.0/
+[bcc testing]: https://github.com/bitcoin-core/bitcoin-devwiki/wiki/24.0-Release-Candidate-Testing-Guide
+[superset rbf]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2013-March/002240.html
+[brqgoo]: https://twitter.com/brqgoo/status/1579216353780957185
+[big msig]: https://blockstream.info/tx/7393096d97bfee8660f4100ffd61874d62f9a65de9fb6acf740c4c386990ef73?expand
+[news65 tapscript limits]: /en/newsletters/2019/09/25/#tapscript-resource-limits
+[lnd 0.15.2-beta]: https://github.com/lightningnetwork/lnd/releases/tag/v0.15.2-beta
+[news221 lnd]: /en/newsletters/2022/10/12/#lnd-v0-15-2-beta
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2022-10-19-newsletter.md:118]:
```suggestion
[news221 lnd]: /zh/newsletters/2022/10/12/#lnd-v0-15-2-beta
```

**Comment by hulatown** [comment]
BTW, may I ask the configuration of the Travis CI check? It will be useful that we can reuse this config in our PrimitivesLane repo and check before submit PRs. Thanks in advance! @bitschmidty

**Comment by bitschmidty** [comment]
@hulatown you should be able to run `make production` which should run the same as CI. 

The issues that I found [here](https://github.com/bitcoinops/bitcoinops.github.io/pull/912#pullrequestreview-1161878292) were from manual checking myself. I open each link in the newsletter and 1. make sure it works and 2. make sure if it is to a newsletter item, that the link uses zh, if it is available.

**Review by bitschmidty — APPROVED** [review]
ACK, thanks for the translation @hulatown !

**Pull request merged**